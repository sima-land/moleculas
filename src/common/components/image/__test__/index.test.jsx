import React from 'react';
import { Image } from '..';
import { mount } from 'enzyme';

describe('<Image />', () => {
  it('should renders correctly', () => {
    const wrapper = mount(
      <Image
        src='www.aaa.com'
        width={23}
        height={34}
      />
    );

    expect(wrapper).toMatchSnapshot();
  });
});
