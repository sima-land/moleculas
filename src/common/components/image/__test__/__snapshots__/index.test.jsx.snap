// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Image /> should renders correctly 1`] = `
<Image
  height={34}
  src="www.aaa.com"
  width={23}
>
  <div
    className="root"
  >
    <img
      height={34}
      src="www.aaa.com"
      width={23}
    />
  </div>
</Image>
`;
